docs/proposed-api.md

```diff
@@ -75,17 +77,19 @@ metadata:
 spec:
   peering:
     vpc-1:
-      ips:
-        - cidr: 10.1.1.0/24
-        - not: 10.1.1.42/32 # that's just a syntactic sugar to avoid enumerating all the subnets explicitly
-      as:
-        - 192.168.1.0/24
-        - not: 192.168.1.7/32 # that's just a syntactic sugar to avoid enumerating all the subnets explicitly
+      expose:
+        - ips:
+            - cidr: 10.1.1.0/24
+            - not: 10.1.1.42/32 # that's just a syntactic sugar to avoid enumerating all the subnets explicitly
+          as:
+            - 192.168.1.0/24
+            - not: 192.168.1.7/32 # that's just a syntactic sugar to avoid enumerating all the subnets explicitly
     vpc-2:
-      ips:
-        - cidr: 10.1.1.0/24
-      as:
-        - 192.168.2.0/24
+      expose:
+        - ips:
+            - cidr: 10.1.1.0/24
+          as:
+            - 192.168.2.0/24
```

Comment:
One thing I find confusing: Why does `ips:` have `cidr:`, but addresses for `as:` are provided without any additional field name?

Reply:
Yeah, missed from the original version, good catch!